envconfig/config.go

```diff
@@ -190,6 +190,14 @@ func LoadConfig() {
 		)
 	}
 
+	additionalOrigins := []string{
+		"app://*",
+		"file://*",
+		"chrome-extension://*",
+	}
+
+	AllowOrigins = append(AllowOrigins, additionalOrigins...)
```

Comment:
Thoughts on appending directly to the AllowOrigins rather than creating the interim slice?

```
AllowOrigins = append(AllowOrigins, 
                "app://*",
		"file://*",
		"chrome-extension://*",
		"tauri://",
)
```